### ISSUE TITLE
Photos additional information doesn't show when Spectrum is active for the Photos app

### ISSUE THREAD

**Issue by SebastianRasch** [Skittyblock/Spectrum#2]
Normally, you can swipe up on pictures in the Photos app to show geo data and additional information but this whole section is missing when Spectrum is active.

**Comment by Skittyblock** [comment]
You're right. This is an interesting bug, and I've got no clue what causes it.
